lib/new_relic/agent/agent.rb

```diff
@@ -97,7 +97,7 @@ def init_event_handlers
         @monitors = Monitors.new(@events)
         @error_collector = ErrorCollector.new(@events)
         @harvest_samplers = SamplerCollection.new(@events)
-        @javascript_instrumentor = JavascriptInstrumentor.new(@events)
+        @javascript_instrumentor = JavaScriptInstrumentor.new(@events)
```

Comment:
This appears to be thing local to the project. But, if it's a public api, i'll gladly drop.

Reply:
You're correct, this is local to the project and renamed in the class definition. Let's keep this change.